manageiq-operator/pkg/helpers/miq-components/generators_test.go

```diff
@@ -0,0 +1,15 @@
+package miqtools
+
+import (
+	"testing"
+)
+
+func TestRandomBytes(t *testing.T) {
+	expected := 10
+	bytes := randomBytes(expected)
+	actual := len(bytes)
+
+	if actual != expected {
+		t.Fatalf(`Expected %d bytes, but got %d bytes.`, expected, actual)
```

Comment:
Spacing here seems bigger than what you used previously (4 instead of 2).

Reply:
Also, wouldn't this (very infrequently) fail when `randomBytes` produces the same as actual?  Or is that not mathematically possible?

Reply:
Go uses tabs, and GitHub doesn't render the first tab properly.

Notice that actual is the length of the array, which should always be 10

Reply:
Okay, yeah, viewed it without the "diff view", and yeah it looks fine.  That is annoying, but good to know.

And I see now I was just not understanding the test.  Whoops.